File: backend/Api.Game/Models/RectangleModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Api.Game.Models
{
    public class RectangleModel : CanvasObjectModel
    {
        public int Height { get; set; }
        public int Width { get; set; }
        public int ColorInHex { get; set; }

        public RectangleModel() : base(CanvasObjectType.Rectangle)
        {
        }
    }
}

File: pixiprototype/src/app/game/playground/_models/canvas-objects/rectangle.model.ts
import { CanvasObject } from './canvas-object.model';
import { CanvasObjectType } from './canvas-object.type';

export class RectangleModel extends CanvasObject {
    height = 0;
    width = 0;
    colorInHex = 0x00000;

    constructor() {
        super(CanvasObjectType.Rectangle);
    }
}
